### ISSUE TITLE
Cannot upload backup from Drive to Home assistant

### ISSUE THREAD

**Issue by 2l8baby** [sabeechen/hassio-google-drive-backup#279]
I am trying to download a Home Assistant backup to do a WIPE and Restore.  I can do a Home Assistant WIPE and RESTORE but not a backup from Google Drive.

I attached logs.  Any help would be appreciated.
John

I am running the following:
Home Assistant 0.116.2
Supervisor: 247
Ubuntu 20.04.1/Docker 19.03.11

[home-assistant-google-drive-backup(1).log](https://github.com/sabeechen/hassio-google-drive-backup/files/5374527/home-assistant-google-drive-backup.1.log)

**Comment by PremiumUsername** [comment]
I'm using HA OS on a RPI3. I have the same problem. Uploads to HA fail. I can download though.

**Comment by Jabol86** [comment]
I have the same problem on rpi3 HA in docker

**Comment by dartfrogdk** [comment]
same here, seems like adguad is messings things up for me, it is not a proper backup

**Comment by sabeechen** [comment]
Looks like the new version of the supervisor requires uploads to have a specific mime-type.   I can see in the supervisor logs on my personal instance when I try to upload a snapshot that it complains:
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/aiohttp/web_protocol.py", line 418, in start
    resp = await task
  File "/usr/local/lib/python3.8/site-packages/sentry_sdk/integrations/aiohttp.py", line 110, in sentry_app_handle
    reraise(*_capture_exception(hub))
  File "/usr/local/lib/python3.8/site-packages/sentry_sdk/_compat.py", line 55, in reraise
    raise value
  File "/usr/local/lib/python3.8/site-packages/sentry_sdk/integrations/aiohttp.py", line 100, in sentry_app_handle
    response = await old_handle(self, request)
  File "/usr/local/lib/python3.8/site-packages/aiohttp/web_app.py", line 458, in _handle
    resp = await handler(request)
  File "/usr/local/lib/python3.8/site-packages/aiohttp/web_middlewares.py", line 119, in impl
    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/security.py", line 132, in system_validation
    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/security.py", line 194, in token_validation
    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/utils.py", line 60, in wrap_api
    answer = await method(api, *args, **kwargs)
  File "/usr/src/supervisor/supervisor/api/snapshots.py", line 191, in upload
    reader = await request.multipart()
  File "/usr/local/lib/python3.8/site-packages/aiohttp/web_request.py", line 588, in multipart
    return MultipartReader(self._headers, self._payload)
  File "/usr/local/lib/python3.8/site-packages/aiohttp/multipart.py", line 530, in __init__
    self._boundary = ('--' + self._get_boundary()).encode()
  File "/usr/local/lib/python3.8/site-packages/aiohttp/multipart.py", line 622, in _get_boundary
    assert mimetype.type == 'multipart', (
AssertionError: multipart/* content type expected
```

Its not part of their documentation.  The supervisor changes its API on a whim, situations like this are unavoidable.
I'll fix this in an update soon.

**Comment by sabeechen** [comment]
I just released the addon version 0.102.0, which includes a fix for this.  Please let me know or reopen this issue if you run into any trouble with the new version.
Also just a reminder, to make Home Assistant see the new version right away go to "Supervisor" > "Add-on Store" > Three dots in the top right corner > "Reload"
